### ISSUE TITLE
Cannot connect to https://zen-browser.app/ [Secure Connection Failed] due to ISP blocking

### ISSUE THREAD

**Issue by 035059** [zen-browser/www#295]
### Error

Attempting to open the Zen-browser homepage or other subpages through Zen, yielded the following error:

```
An error occurred during a connection to zen-browser.app. SSL received a record that exceeded the maximum permissible length.

Error code: SSL_ERROR_RX_RECORD_TOO_LONG
```
Auto-updates were also non-functional, and no other browsers could open the page, some reporting other errors.

Looking into the issue, I tried to access the http (non-s) version of the site, after bypassing the non-secure warning I was greeted by a "this site as been blocked message", and a "Plume" logo.

### Work-around

It appears my ISP (Cogeco) is using a service called Plume, which blocked access to zen-browser.app, for "phishing and malware".

I was able to allow access to the site in the "MyCogeco" app, under the "Guard Events" tab (god I hate home networking in 2024), so this may serve as a work around for others who are experiencing the same issue. I noticed similar presentations (though different errors) in #289 and #294.

I can now access the site without issues, and auto-update is working.

### Report Reason

Reporting this as an issue as this is the first site my ISP has blocked, my first guess is this is something related to the config of the site, but I'm no web-dev. 

Hopefully this will help point others with same issue in the correct direction, and gives the devs something to point to if others are encountering the same error.

**Comment by sarisman84** [comment]
Glad it wasn't just me. Ill wait for the fix since i can just download the latest version of the browser on github. Hopefully this can get fixed.

**Comment by NOCanoa** [comment]
Are u using ur isps DNS?

**Comment by 035059** [comment]
> Are u using ur isps DNS?

No, I don't. The IP always properly resolved, even when blocked.

From my understanding the issue is a result of a service my ISP uses called [Plume](https://www.plume.com/). I submitted a ticket a ticket to my ISP to try and get the to remove it from their list, right after I submited the ticket here, but just checking again now, their service is still blocking the site.

The bigger issue is this likely means more users will be affected, as Plume is a partner of over 240 ISPs including Comcast, Charter, Qualcomm, Cablevision, Shaw Communications, Bell Canada, and Liberty Global.(according to wikipedia)

I believe it is due to the similarity to the site [zen-browser.com](zen-browser.com), which shows up on archive.org as early as 2019, and I can find downloads for the program dating as early as 2018.
